homework-solutions/ts/hw-14/task-1.ts

```diff
@@ -0,0 +1,38 @@
+
+//1. Создайте дженерик функцию wrapInArray, которая принимает значение любого типа и возвращает его в виде массива этого типа.
+    function wrapInArray<T>(result: T):[T]{
+      return [result];
+    }
+
+    const numberArray = wrapInArray(5); // [5]
+    const stringArray = wrapInArray('Hello'); // ['Hello']
+
+    console.log(numberArray);
+    console.log(stringArray);
+
+//2. Создайте дженерик функцию getLastElement, которая принимает массив элементов типа T, и возвращает последний элемент (типа T).
+
+    function getLastItem<T>(arr: T[]): T | unknown  {
```

Comment:
```suggestion
    function getLastItem<T>(arr: T[]): T | undefined {
```
Тут либо Т либо массив пустой, и тогда вернется андефайнд